### PR TITLE
Cuibackofficepages

### PR DESCRIPTION
**Pull request by rajguptaH**
Testing Is Done By Ritesh And Everything Is Working Fine.
[testing.xlsx](https://github.com/smartdusttechnologies/TestingAndCalibrationLabs/files/9333691/testing.xlsx)

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TestingAndCalibrationLabs.Business.Common;
+using TestingAndCalibrationLabs.Business.Core.Model.UiControl;
+
+namespace TestingAndCalibrationLabs.Business.Core.Interfaces.Cui
```
Comment (Rishirakish) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs:7]: remove cui word from namespace.

Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs:7]: remove cui word from namespace.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public interface IUiControlService
+    {
+        UiControlModel Get(int id);
+
+        List<UiControlModel> GetAll();
+        RequestResult<int> Edit(int id, UiControlModel conModel);
+        RequestResult<int> Create(UiControlModel conModel);
+        bool Delete(int id);
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs:16]: put meaningful comments at interface level and it's methods.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageControlService.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TestingAndCalibrationLabs.Business.Common;
+using TestingAndCalibrationLabs.Business.Core.Model.UiPageControl;
+
+namespace TestingAndCalibrationLabs.Business.Core.Interfaces.Cui
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageControlService.cs:7]: remove cui word

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageControlService.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+{
+    public interface IUiPageControlService
+    {
+        List<UiPageControlModel> GetAll();
+        UiPageControlModel GetById(int id);
+        RequestResult <int> Create(UiPageControlModel pageControl);
+        RequestResult <int> Update(int id , UiPageControlModel pageControl);
+        bool Delete(int id);
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageControlService.cs:15]: proper comments

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageService.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+namespace TestingAndCalibrationLabs.Business.Core.Interfaces.Cui
+{
+    public interface IUiPageService
+    {
+        List<UiPageModel> GetAll();
+        List<PageModel> GetPage();
+        List<DataModel> GetData();
+        List<ControlModel> GetControl();
+        List<ValModel> GetVal();
+        List<MetadataModel> GetMetadata();
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiPageService.cs:17]: what are these methods. Name is not making sense and also discuss with me the use of these methods.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ControlModel.cs [hunk 6]
```diff
@@ -0,0 +1,8 @@
+﻿namespace TestingAndCalibrationLabs.Business.Core.Model.Helper
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ControlModel.cs:1]: why helper word is these in namespace.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ControlModel.cs [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+{
+    public class ControlModel
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ControlModel.cs:3]: name of all the models should be changed. discuss with me

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/PageModel.cs [hunk 8]
```diff
@@ -0,0 +1,8 @@
+﻿namespace TestingAndCalibrationLabs.Business.Core.Model.Helper
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/PageModel.cs:1]: Helper word

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ValModel.cs [hunk 9]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TestingAndCalibrationLabs.Business.Core.Model.Helper
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Helper/ValModel.cs:5]: Helper word

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/UiControl/UiControlModel.cs [hunk 10]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TestingAndCalibrationLabs.Business.Core.Model.UiControl
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/UiControl/UiControlModel.cs:5]: UiControl word

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Cui/UiPageControlRepo.cs [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+                                where id = @Id ";
+            using IDbConnection db = _connectionFactory.GetConnection;
+            db.Execute(query, new { isDeleted = true, Id = id });
+            return true;
+        }
+
+        public List<UiPageControlModel> GetAll()
+        {
+            using IDbConnection db = _connectionFactory.GetConnection;
+           // string Query = "Select* From[UiPageMetadata] where IsDeleted = 0 SELECT UiPageMetaData.Id, UiPageType.Name, UiControlType.Name FROM((UiPageMetadata INNER JOIN UiPageType ON UiPageMetadata.Id = UiPageType.Id)INNER JOIN UiControlType ON UiPageMetadata.Id = UiControlType.Id);";
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Cui/UiPageControlRepo.cs:42]: what is this comment.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs (hunk 1 above)
Comment (rajguptaH) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/Cui/IUiControlService.cs:7]: for interface only or for all the files that i have maked under cui folder..

**Pull request closed without merging**